Add missing arg value in multi-app-template.md

stuartleeks opened the pull request:
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

<!--Please explain the changes you've made-->

Added a missing arg placeholder for `dapr stop -f` example

## Issue reference

<!--Please reference the issue this PR will close: #[issue number]-->

stuartleeks committed:
Add missing arg value in multi-app-template.md

Signed-off-by: Stuart Leeks <stuartle@microsoft.com>

stuartleeks commented:
I see that the `validate-links` check has failed, but I haven't made any links edits in this PR 😄

hhunter-ms commented:
@stuartleeks hm I'm not sure why the link isn't working - it works for me. try bookending the 'broken' links with these:

> < !-- IGNORE_LINKS -- >
> < !-- END_IGNORE -- >

stuartleeks committed:
Update broken links

https://account.kubemq.io/login/register is redirected to https://account.kubemq.io/auth/signup
Signed-off-by: Stuart Leeks <stuartle@microsoft.com>

stuartleeks commented:
@hhunter-ms - it turns out that the broken link has a 301 redirect, so I've updated the links to the new URL

hhunter-ms commented on daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md:
```diff
@@ -51,7 +51,7 @@ This component supports both **input and output** binding interfaces.
 
 {{% codetab %}}
-1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/login/register](https://account.kubemq.io/login/register) and register for a key.
+1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/auth/signup](https://account.kubemq.io/auth/signup) and register for a key.
```
```suggestion
<!-- IGNORE_LINKS -->
1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/auth/signup](https://account.kubemq.io/auth/signup) and register for a key.
<!-- END_IGNORE -->
```

hhunter-ms commented on daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md:
```diff
@@ -64,7 +64,7 @@ You can then interact with the server using the client port: `localhost:50000`
 
 {{% codetab %}}
-1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/login/register](https://account.kubemq.io/login/register) and register for a key.
+1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/auth/signup](https://account.kubemq.io/auth/signup) and register for a key.
```
```suggestion
<!-- IGNORE_LINKS -->
1. Obtain KubeMQ Key by visiting [https://account.kubemq.io/auth/signup](https://account.kubemq.io/auth/signup) and register for a key.
<!-- END_IGNORE -->
```

hhunter-ms committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md

Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>

hhunter-ms committed:
Update daprdocs/content/en/reference/components-reference/supported-bindings/kubemq.md

Signed-off-by: Hannah Hunter <94493363+hhunter-ms@users.noreply.github.com>
